tests/conftest.py

```diff
@@ -13,6 +13,12 @@
 pytest_plugins = 'sphinx.testing.fixtures'
 
 
+@pytest.fixture(autouse=True)
+def env_setup(monkeypatch):
+    monkeypatch.setenv('READTHEDOCS', 'True')
+    monkeypatch.setenv('TESTING_ENVIRONMENT', 'True')
```

Comment:
You can remove this one now, I guess

Reply:
Ohh... I forget about that one.
Removed. :+1: